packages/theming/apple-theme/src/appleTypography.macos.ts

```diff
@@ -0,0 +1,57 @@
+import { FontSize, FontSizes, FontWeightValue, Typography, Variants } from '@fluentui-react-native/theme-types';
+
+/**
+ * The Typography is designed to match the styles defined in the Apple HIG:
+ * https://developer.apple.com/design/human-interface-guidelines/macos/visual-design/typography/
+ * These mappings and variants are subject to change as we moved to a unified cross platform Fluent typography ramp
+ */
+
+export function appleTypography(): Typography {
+  const appleDict = {
```

Comment:
Is there a bug tracking these issues coming from native components? Probably worth follow up to make it robust so we never have issues with it again instead of tweaking these values for every release :)

Reply:
#598 filed, adding as a comment as well